### PR TITLE
#5071 - Added Easyform scala API and demo.

### PR DESCRIPTION
**Pull request by michalgce**

### CODE REVIEW

File: kernel/scala/src/main/java/com/twosigma/beaker/scala/easyform/EasyForm.scala [hunk 1]
```diff
@@ -0,0 +1,111 @@
+package com.twosigma.beaker.scala.easyform
```
Comment (scottdraves) [inline kernel/scala/src/main/java/com/twosigma/beaker/scala/easyform/EasyForm.scala:17]: all files need banner

**Comment by scottdraves** [comment]
thanks, good work, looks like the right idea, but I didn't see how to access the values in the form.
can you add a cell to be run that works?  i tried this:
screen shot 2017-05-31 at 11 27 36 am

File: kernel/scala/src/main/java/com/twosigma/beaker/scala/easyform/EasyForm.scala (hunk 1 above)
Comment (michalgce) [reply kernel/scala/src/main/java/com/twosigma/beaker/scala/easyform/EasyForm.scala:17]: 👍

File: kernel/base/src/main/java/com/twosigma/beaker/chart/xychart/plotitem/XYGraphics.java [hunk 2]
```diff
@@ -92,7 +92,7 @@ public void setX(List<Object> xs) {
   }
 
   public List<Number> getX() {
-    if (xs == null) {
+    if (xs == null || xs.isEmpty()) {
```
Comment (scottdraves) [inline kernel/base/src/main/java/com/twosigma/beaker/chart/xychart/plotitem/XYGraphics.java:95]: why was this added?  doesn't seem right.  an empty xs array seems like an error.

**Comment by scottdraves** [comment]
🐬
